src/views/LostItems/LostItemsListContainer/components/LostItemsList/components/ActualCostConfirmModal/ActualCostConfirmModal.test.js

```diff
@@ -0,0 +1,138 @@
+  doNotBillConfirmText: 'ui-users.lostItems.modal.doNotBill.confirm.text',
+  additionalInformationForStaff: 'ui-users.lostItems.modal.additionalInformationForStaff',
+  additionalInformationForPatron: 'ui-users.lostItems.modal.additionalInformationForPatron',
+  confirmBillTitle: 'ui-users.lostItems.modal.bill.confirm.title',
+  confirmDoNotBillTitle: 'ui-users.lostItems.modal.doNotBill.confirm.title',
+  confirmButton: 'ui-users.lostItems.modal.button.confirm',
+  keepEditing: 'ui-users.lostItems.modal.button.keepEditing',
+};
+const testIds = {
+  actualCost: 'actualCost',
+  confirmActualCost: 'confirmActualCost',
+  keepEditingActualCost: 'keepEditingActualCost',
+};
+
+describe('ActualCostConfirmModal', () => {
+  describe('when actual cost type equals "bill"', () => {
+    beforeEach(() => {
+      render(
+        <ActualCostConfirmModal
+          {...initialProps}
+        />
+      );
+    });
+
+    it('should render modal window', () => {
+      expect(screen.getByTestId(testIds.actualCost)).toBeVisible();
+    });
+
+    it('should have correct title', () => {
+      expect(screen.getByText(messageIds.confirmBillTitle)).toBeVisible();
+    });
+
+    it('should render confirm bill message', () => {
+      expect(screen.getByText(messageIds.billConfirmText)).toBeVisible();
+    });
+
+    it('should render "additionalInformationForStaff" message', () => {
+      expect(screen.getByText(messageIds.additionalInformationForStaff)).toBeVisible();
+    });
+
+    it('should render "additionalInformationForPatron" message', () => {
+      expect(screen.getByText(messageIds.additionalInformationForPatron)).toBeVisible();
+    });
+
+    describe('footer', () => {
+      it('should render "Confirm" button label', () => {
+        expect(screen.getByText(messageIds.confirmButton)).toBeVisible();
+      });
+
+      it('should trigger "setActualCostConfirmModal" and "setActualCost" with correct data', () => {
+        const confirmButton = screen.getByTestId(testIds.confirmActualCost);
+
+        fireEvent.click(confirmButton);
+
+        expect(initialProps.setActualCostConfirmModal).toHaveBeenCalledWith(ACTUAL_COST_CONFIRM_MODAL_DEFAULT);
+        expect(initialProps.setActualCost).toHaveBeenCalledWith(ACTUAL_COST_DEFAULT);
+      });
+
+      it('should render "keepEditing" button label', () => {
+        expect(screen.getByText(messageIds.keepEditing)).toBeVisible();
+      });
+
+      it('should trigger "setActualCostConfirmModal" and "setActualCostModal" with correct data', () => {
+        const keepEditingButton = screen.getByTestId(testIds.keepEditingActualCost);
+
+        fireEvent.click(keepEditingButton);
+
+        expect(initialProps.setActualCostConfirmModal).toHaveBeenCalledWith(ACTUAL_COST_CONFIRM_MODAL_DEFAULT);
+        expect(initialProps.setActualCostModal).toHaveBeenCalledWith({
+          isOpen: true,
+        });
+      });
+    });
+  });
+
+  describe('when actual cost type equals "doNotBill"', () => {
```

Comment:
Could you please check can we check that additionalInformationForPatron are absent

Reply:
Added initial test